src/bundle.js

```diff
@@ -68,6 +68,7 @@ export async function bundle(input) {
     entryPoints: [input],
     bundle: true,
     write: false,
+    format: 'esm',
```

Comment:
From [esbuild docs](https://esbuild.github.io/api/#format-esm):
>The esm format stands for "ECMAScript module". It assumes the environment supports import and export syntax.

We don't support import and export syntax, do you know if any issues will arise from this change?